pa

```diff
@@ -99,7 +99,11 @@ pw_list() {
 }
 
 git_add_and_commit() {
-    if $git_enabled; then git add "$1" && git commit -qm "$2"; fi
+    git add "$1" ||
+        die "couldn't git add $1"
```

Comment:
all of the other error messages are Capitalized (except when it starts with the name of the utility like `age`).
I'm not against lowercase errors - in fact I think it's better for aligning with unix convention - but I believe it has to be consistent across the project.

Reply:
ok :D https://github.com/biox/pa/pull/45/commits/d575d81a8883b98ecebb189d741d78a45448ae4e

Reply:
radical, I respect that. there are contribs left, but they can be handled separately